### PR TITLE
Moved top table to a new top component

### PR DESCRIPTION
**Pull request by agnlez**

### CODE REVIEW

**Comment by agnlez** [comment]
@hectoruch I have separated Top "table" to a new component... Things were becoming complicated. Take a look please.

**Comment by hectoruch** [comment]
All works fine! @andresgnlez 👍

File: scripts/components/factsheets/top.component.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  _parseData() {
+    let baseUrlLink;
+
+    switch (this.targetLink) {
+    case 'place':
+      baseUrlLink = '/factsheet-place.html?nodeId=';
+      break;
+    case 'actor':
+      baseUrlLink = '/factsheet-actor.html?nodeId=';
```
Comment (nerik) [inline scripts/components/factsheets/top.component.js]: I think you can make this simpler by simply using a template string (ie '/factsheet-${this.targetLink}.html?nodeId=')

File: scripts/pages/factsheet-place.page.js [hunk 2]
```diff
@@ -26,25 +27,34 @@ const defaults = {
...
-    type: 'top'
-  });
+
+  new Top({
+    el: document.querySelector('.js-top-trader'),
+    data: data.top_traders,
+    targetLink: 'actor',
+    title: 'top traders',
+    unit: '%'
+  }).render(renderCallback);
```
Comment (nerik) [inline scripts/pages/factsheet-place.page.js]: I don't understand the need for a callback, as there is nothing async in render()

Comment (agnlez) [reply scripts/pages/factsheet-place.page.js]: There isn't yeah. But this way you allow the component to do something after call its render function and the component doesn't care about what is running, it just does it if it exists.

Comment (nerik) [reply scripts/pages/factsheet-place.page.js]: I still don't see the point. You do :
```
new Top (...)
new Top (...)
this.el.parentElement.classList.toggle (...)
```
it's functionally the same without the unnecessary tight coupling between the component and its parent.

File: scripts/utils/changeNumber.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ export default (number, type)  => {
 
   if(number != null){
     if(number > 0) {
-      if(type === 'top') {
+      if(type === 'percentage') {
```
Comment (nerik) [inline scripts/utils/changeNumber.js:6]: Since type can only be undefined or 'percentage', can we replace this with an isPercentage boolean?

Comment (agnlez) [reply scripts/utils/changeNumber.js:6]: I'll work on this in another branch, I want to make several changes here.

**Pull request merged**